docs/faq.md

```diff
@@ -22,4 +22,20 @@ time="2018-03-13T12:08:31Z" level=fatal msg="async engine stopped: error sending
 
 After checking on the health of the kube-system pods, kube-dns was not working. This caused osba 
 to be unable to connect to its Redis instance. The problem was resolved by deleting the
-kube-dns pods, so that Kubernetes would recreate them.
\ No newline at end of file
+kube-dns pods, so that Kubernetes would recreate them.
+
+## I don't see all the services
+
+The helm chart is configured to set the minimum stability level of the broker to `preview`,
```

Comment:
Can we lead into this with a brief one or two sentence explanation (in case it isn't well-understood by the reader) that the broker's services are provided by a series of modules and those modules all have their own level of stability? It's hard to put myself in the shoes of someone who doesn't already know OSBA inside and out, but I think one could infer all that from this paragraph if you're good at reading between the lines, but we should be explicit about it.

This bit could throw people off:

> the minimum stability level of the broker

Since it sounds like it refers to the broker itself, rather than constituent pieces of it.

Reply:
Same applies in other places where this same text appears, obviously.